storage/mem/user_storage.go

```diff
@@ -125,3 +120,13 @@ func (us *UserStorage) ImportJSON(data []byte) error {
 
 // Close does nothing here.
 func (us *UserStorage) Close() {}
+
+func randUser() model.User {
+	return model.User{
```

Comment:
This is not related to the topic of this PR, but I would really appreciate an option to make this output predictable - basically, supply a mocked user to the storage when it's initialised.